### ISSUE TITLE
Enhancement to Search function

### ISSUE THREAD

**Issue by GiselleSterry** [nbnuk/nbnatlas-issues#216]
Firstly, congratulations! I’m still finding my way around the new system but I’m pleased that the new system allows me to zoom in to quite a good extent on records. Useful having locations on the map too as it helps to visualise where the record actually is and handy to swap between different map tiles.

The one part of the functionality that I am missing is the ability to search species by using a five letter abbreviation of the scientific name, e.g. Syrib as an abbreviation of the hoverfly Syrphus ribesii. When I enter this now as a search term I come up with no results and presumes I was searching for the location Syria.

Will this functionality be returning? It’s really handy for me, saves time typing and is especially useful when entering terms straight from my abbreviated field notes.

**Comment by CEJohnston** [comment]
Similar to #168

**Comment by sophiathirza** [comment]
Does the UKSI contain this abbreviations, if so, could we include them in the search.

**Comment by c-van-dierkson** [comment]
#168 is now closed, has this been addressed by autofill options? Can we close this one? @sophiathirza

**Comment by sophiathirza** [comment]
@c-van-dierkson no, this is about users being able to search for species using a five letter code. The code is in the UKSI but we don't export it. The code would need to be added to the BIE search index.
